mopidy_musicbox_webclient/static/js/functionsvars.js

```diff
@@ -356,7 +353,7 @@ function resultsToTables(results, target, uri) {
     tableid = "#" + tableid;
     $(target).html(html);
     $(target).attr('data', uri);
-    //retreive albumcovers
+    //retrieve albumcovers
     for (i = 0; i < coversList.length; i++) {
         getCover(coversList[i][0], target + '-cover-' + coversList[i][1], 'small');
```

Comment:
An album header should be shown when they are more than one tracks form the same album. It should not require all the tracks. I believe it has this behaviour to avoid looking stupid (and twice as long) when it is just a big list of random songs.